src/blueprint/links.rs

```diff
@@ -0,0 +1,37 @@
+use crate::config::Link;
+use crate::directive::DirectiveCodec;
+use crate::valid::{Valid, ValidationError, Validator};
+
+// Wrap the Vec<Link> is necessary to implement TryFrom
+#[allow(dead_code)]
```

Comment:
This is necessary as `(Vec<Link>)` is not used anywhere in the code.

Optionally, we can remove it and change `Links` to be an empty struct – `pub struct Links` – or a unit type – – `pub struct Links(())` –.